ref/general/microshed.adoc

```diff
@@ -17,14 +17,14 @@ In integration testing, you can put together individual pre-tested modules and t
 
 == What challenges does MicroShed resolve
 
-An application might run differently in production as compared to test/dev environment. This can cuse a test to fail in production though it passed in  dev/test. You can address this issue by testing your application in the same container that you’ll use in production.
+An application might run differently in production as compared to test/dev environment. This can cause a test to fail in production though it passed in  dev/test. You can address this issue by testing your application in the same container that you’ll use in production.
```

Comment:
For "test/dev" environment and "dev/test", make sure to verify whether that is standard terminology.

Reply:
Okay